recipes/autoflake/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "autoflake" %}
+{% set version = "1.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "c103e63466f11db3617167a2c68ff6a0cda35b940222920631c6eeec6b67e807" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  entry_points:
+    - autoflake = autoflake:main
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - pip
+    - pyflakes >=1.1.0
```

Comment:
Is this needed as a `host` dependency? This section only needs to contain the packages needed to run the `setup.py` file:
```suggestion
```